packages/FullSimulation/MiniAOD/make_MiniAOD.py

```diff
@@ -164,20 +164,24 @@ def exit_runhosterr(runmode,hostname):
     runshellfile.write("cd "+cwd+"/"+runmode+"/"+sample_name+"/run"+str(ijob)+"/\n")
     runshellfile.write("source /cvmfs/cms.cern.ch/cmsset_default.sh\n")
     runshellfile.write("eval `scramv1 runtime -sh`\n")
-    this_cmsdrivercmd = cmsdrivercmd+" --python_filename run"+str(ijob)+".py --fileout \"file:"+cwd+"/output/"+sample_name+"/"+nametag+"_"+str(ijob)+".root\" --filein \"file:"+inputlines[ijob-1].strip()+"\""
+    this_cmsdrivercmd = cmsdrivercmd+" --python_filename run"+str(ijob)+".py --fileout \"file:"+cwd+"/output/"+sample_name+"/"+nametag+"_"+str(ijob)+".root\" --filein \""+inputlines[ijob-1].strip()+"\""
     runshellfile.write(this_cmsdrivercmd+"\n")
     runshellfile.write("cmsRun run"+str(ijob)+".py &> run"+str(ijob)+".log\n")
     runshellfile.close()
+    os.chmod(runmode+"/"+sample_name+"/run"+str(ijob)+"/run"+str(ijob)+".sh",0o755)
 
     submitshellfile.write("cd "+cwd+"/"+runmode+"/"+sample_name+"/run"+str(ijob)+"\n")
     if (runmode == "MULTICORE"):
-      submitshellfile.write("chmod 755 "+cwd+"/"+runmode+"/"+sample_name+"/run"+str(ijob)+"/run"+str(ijob)+".sh\n")
       submitshellfile.write("nohup "+cwd+"/"+runmode+"/"+sample_name+"/run"+str(ijob)+"/run"+str(ijob)+".sh &\n")
     elif (runmode == "CLUSTER"):
       os.system("cp skeleton/condor.jds "+runmode+"/"+sample_name+"/run"+str(ijob))
       os.system("sed -i 's|###CONDORRUN|executable = run"+str(ijob)+".sh|g' "+runmode+"/"+sample_name+"/run"+str(ijob)+"/condor.jds")
       os.system("sed -i 's|###JOBBATCHNAME|+JobBatchName=\""+sample_name+"\"|g' "+runmode+"/"+sample_name+"/run"+str(ijob)+"/condor.jds")
       submitshellfile.write("condor_submit condor.jds\n")
+      if os.getenv("HOSTNAME") in ["tamsa1","tamsa2"]:
```

Comment:
Can you change these lines to "if (use_SNU == True):" instead?
CLUSTER & SNU = tamsa server

Reply:
done